.github/workflows/on_release_publish.yml

```diff
@@ -161,42 +161,63 @@ jobs:
     env:
       DOCKERHUB_ORG: gospatial
       DOCKERHUB_REPO: tegola
+      BUILD_ARGS: |
+        BUILDPKG=${BUILD_PKG}
+        VER=${VERSION}
+        BRANCH=${GIT_BRANCH}
+        REVISION=${GIT_REVISION}
 
     steps:
     - name: Check out code
       uses: actions/checkout@v3
 
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Login to Docker Hub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKERHUB_USER }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
     - name: Setup env
       uses: ./.github/actions/tegola-setup-env
       with:
         ui: true
 
-    - name: Build and tag Docker container
+    - name: Build and export to Docker
+      uses: docker/build-push-action@v4
+      with:
+        context: .
+        load: true
+        platforms: linux/amd64
+        build-args: ${{ env.BUILD_ARGS }}
+        tags: ${{ env.DOCKERHUB_ORG }}/${{ env.DOCKERHUB_REPO }}:${{ env.VERSION }}
+
+    - name: Test
```

Comment:
Please give this step a better name. At first glance I thought it was leftover test code.

Reply:
Done, I hope new one is better.